docs/sources/reference/commandline/cli.md

```diff
@@ -98,16 +102,16 @@ you can write `docker run -it --name test busybox sh`.
 
 ### Boolean
 
-Boolean options take the form `-d=false`. The value you see in the help text is the
-default value which is set if you do **not** specify that flag. If you specify
-a Boolean flag without a value, this will set the flag to `true`, irrespective
-of the default value.
+Boolean options take the form `-d=false`. The value you see in the help text is
+the default value which is set if you do **not** specify that flag. If you
```

Comment:
We can clarify this by using a restrictive clause and tighten the structure: ->

The value displayed in the help is the default Docker uses if you omit that flag. If you specify a Boolean flag without a value, Docker uses a value of `true` irrespective of the flag's default.